scripts/easy-spinnaker/inst_oes.sh

```diff
@@ -5,13 +5,13 @@
 #################################################################################################################
 ### PLEASE UPDATE THESE WITH THE USERNAME AND CREDENTIALS provided by OpsMX
 #################################################################################################################
-DOCKER_USERNAME=<username>
-DOCKER_PASSWORD=<password>
+DOCKER_USERNAME=USER_NAME_GOES_HERE
+DOCKER_PASSWORD=PASSWORD_GOES_HERE
```

Comment:
The installation will fail if the user does not update this. Do you want to add a check somewhere and fail with a proper message if the user does not update these values?

Reply:
Fixed.